modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -137,23 +137,36 @@ To resolve conflicts on the client side, a `conflictResolutionStrategy` needs to
 
 If none is provided, by default, the `clientVersionWins` strategy is used. This means the SDK will automatically override the server data with the current client data. 
 
-It is very easy to provide a custom conflict resolution strategy: it is a function that will take in the current server data, and client data as parameters, and the return value will be the resolved data. 
+It is very easy to provide a custom conflict resolution strategy. It takes two parameters, `strategies` and `default`.
 
-Here is an examples:
+* strategies - a dictionary object defining the custom behaviour for any mutation you which to specify
```

Comment:
```suggestion
* strategies - a dictionary object defining the custom behavior for any mutation you want to specify
```
not sure about this one

Reply:
`behavior` are we using US spelling in general?

Reply:
yes :smiley_cat: